games/snake/snake_input_console.h

```diff
@@ -63,6 +63,25 @@ void init_termios(int echo)
   fcntl(STDIN_FILENO, F_SETFL, O_NONBLOCK);  /* set non-blocking mode */
 }
 
+/****************************************************************************
+ * Name: deinit_termios
+ *
+ * Description:
+ *   Restore back g_old terminal I/O settings.
+ *
+ * Parameters:
+ *   None
+ *
+ * Returned Value:
+ *   None.
+ *
+ ****************************************************************************/
+
+void deinit_termios(void)
```

Comment:
Isn't it same as reset_termios() ?

Reply:
yes, @eren-terzioglu could you merge into one